src/oasis-nexus/api.ts

```diff
@@ -714,6 +723,7 @@ export const useGetRuntimeEvents: typeof generated.useGetRuntimeEvents = (
               ) {
                 return {
                   ...event,
+                  evm_log_params: event.evm_log_params?.map(fixChecksumAddressInEvmEventParam),
```

Comment:
oh, it's this easy. No need to check event types because 0x addresses only appear in evm.log params. Not in e.g. `body` of consensus_accounts.deposit

Reply:
Indeed.